release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -274,6 +274,18 @@ The `openshiftVersion` parameter of the `AgentServiceConfig` custom resource (CR
 
 For more information see, xref:../scalability_and_performance/routing-optimization.adoc#configuring-ingress-controller-router-liveness-readiness-and-startup-probes[Configuring Ingress Controller (router) Liveness, Readiness, and Startup probes]
 
+[id=ocp-4-11-node-observability-operator]
+==== Node Observability Operator (Technology Preview)
+{product-title} 4.11 introduces the Node Observability Operator in Technology Preview.
+
+Node Observability Operator provides the following features:
+
+* Trigger a CRIO-O and kublet profiling.
+* Deploy Node Observability Agents on selected nodes.
+* Retrive the profiling data files for analysis.
```

Comment:
```suggestion
* Retrieve the profiling data files for analysis.
```

Reply:
Not sure this last feature is worth mentioning : In the tech preview, we're still storing everything locally, on each agent. We're only providing a little manual shell script to retrieve the files. 
Up to you

Reply:
@sherine-k well, then I would remove to avoid confusion.